integration/sql/iface-java/src/main/java/io/openlineage/sql/Quotes.java

```diff
@@ -0,0 +1,43 @@
+/*
+/* Copyright 2018-2023 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.sql;
+
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+public class Quotes {
+  private final String database;
```

Comment:
Could we rename this into `databaseQuoteStyle`, `schemaQuoteStyle`, `nameQuoteStyle` ?

Reply:
Is there any database where quote styles for those are different?

Reply:
Sure, let's hit the road 🚗

Reply:
I disagree with this rename, it's obvious in the context of the class. If anything, rename the class to `QuoteStyle`.